mysql2.gemspec

```diff
@@ -13,9 +13,4 @@ Gem::Specification.new do |s|
 
   s.files = `git ls-files README.md CHANGELOG.md LICENSE ext lib support`.split
   s.test_files = `git ls-files spec examples`.split
-
-  # tests
-  s.add_development_dependency 'rake-compiler', '~> 0.9.5'
-  s.add_development_dependency 'rake', '~> 0.9.3'
-  s.add_development_dependency 'rspec', '~> 2.8.0'
```

Comment:
I suppose for rubygems.org that's a valid point. In my experience, I've never known anyone to audit that list (as given by rubygems) for any purpose. In fact, it seems that if someone really cares about the dependency list they'd have to audit the dependencies as given by bundler, rather than rubygems.

I moved these because the gemspec can't express a variety of conditions that the Gemfile can, and the use of both just adds cognitive overhead. Does that sound reasonable?